docs/transaction/create-sign-transaction.md

```diff
@@ -0,0 +1,21 @@
+### Create Unsigned Transaction
+
+This section guides you through the process of creating a new singed transaction using the WalletInfra.
+
+```ts
+import {
+  ITransactionSigned,
+  TransactionTypeKeys,
+} from '@models/transaction.models';
+import { WalletInfra } from './wallet-infra';
+
+const walletInfra = new WalletInfra();
+
+const mySingedTransaction: ITransactionSigned = {
+  transactionType: 'signed',
```

Comment:
this can be used as `transactionType: TransactionTypeKeys.SIGNED` to better looking example